tikv-client/src/main/java/com/pingcap/tikv/key/Key.java

```diff
@@ -116,6 +116,15 @@ public Key next() {
     return toRawKey(Arrays.copyOf(value, value.length + 1));
   }
 
+  /**
+   * nextPrefix key will be key with next available rid.
```

Comment:
it will be better if you can provide some examples

Reply:
done